cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -792,6 +810,7 @@ def setup_tokens(token, username, user):
         stream.write('{0},{1},{2}'.format(token, username, user))
 
 
+@retry(times=2, delay_secs=20)
```

Comment:
Could we set the delay to 10 seconds and do it 6 times? I think a minute wait should be appropriate